base/multimedia.jl

```diff
@@ -94,9 +94,10 @@ your images to be displayed on any PNG-capable `AbstractDisplay` (such as IJulia
 to `import Base.show` in order to add new methods to the built-in Julia function
 `show`.
 
-The default MIME type is `MIME"text/plain"`. There is a fallback definition for `text/plain`
-output that calls `show` with 2 arguments. Therefore, this case should be handled by
-defining a 2-argument `show(io::IO, x::MyType)` method.
+To customize how an object of type `T` is shown in text-based outputs, it is highly recommended
+to overload `show(::IO, ::MIME"text/plain", ::T)` rather than [`show(::IO, ::T)`](@ref show).
+Container types should call `show(io, MIME"text/plain"(), x)` for elements `x`
```

Comment:
This only applies to the 3-argument `show` method for the container type.

Both here and in the other doc string I think we should use expressions such as "human readable" vs. "parseable" to clarify the difference. As it is, it almost sounds like 2-argument show should never be used.

Reply:
Thanks for reviewing this. Does the new commit address your concern?